packages/nimble-components/src/wafer-map/data-types/WaferMapColorsScale.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * WaferMapColorsScale
```

Comment:
Could you add a bit documentation for each of these classes? It can be basic like "Represents the color scale for a wafer map component" but it should be a sentence rather than just repeating the name of the class.

Reply:
Documentation for public properties is also helpful but not strictly required.

Reply:
I added more descriptions